app/models/bulk_tax_return_update.rb

```diff
@@ -25,7 +27,7 @@ class BulkTaxReturnUpdate < ApplicationRecord
   belongs_to :tax_return_selection
   belongs_to :assigned_user, class_name: "User", optional: true
 
-  enum status: TaxReturnStatus::STATUSES, _prefix: :status
+  enum status: TaxReturnStateMachine.states, _prefix: :status
```

Comment:
if we're understanding this line (we = me + @tgrathwell) it looks like it changes the enum definition from `{ intake_before_consent: 100 ...` to `["intake_before_consent", ...` which seems like it won't have the same meaning for btru's before and after this commit.

are we trying to preserve this enum for Max? (i'm assuming so since this adds the redundant `state` column) if so we should probably leave the enum definition the same (as in `tax_return.rb`)

Reply:
yes that's true -- it is correctly maintained in tax_return model, but got changed incorrectly here. I will revert it. Also, the goal is to eventually drop status, but only once we've done the migrations to state and informed Max of the transition. We have to take a stepped approach.